recipes/jbig/Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- Makefile	2014-03-27 14:47:15.000000000 -0400
++++ Makefile	2015-11-06 14:14:48.000000000 -0500
+@@ -4,7 +4,7 @@
+ CC = gcc
+ 
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
++CFLAGS = -O2 -W -fPIC
```

Comment:
I added my question on the [commit]( https://github.com/conda/conda-recipes/commit/5609b2a1b4bd7fd8fe90f01747a48e7ece53b947#commitcomment-17132447 ) that added the patch too.